### PR TITLE
fix RGB feedback for RGB adapter

### PR DESCRIPTION
**Pull request by JanJaapKo**
made it more inline with the RGBW adapter

### CODE REVIEW

**Comment by JanJaapKo** [comment]
@stas-demydiuk  The 'color_light.py' is now equal to the 'color_colortemp.py' except for the creation of the Domoticz device. One way to solve this could be via the capabilities as they are created in the adapter. However, the adapters could also be made equal. Other way to avoid duplication would be to move the generic methods `get_color_value` and `get_rgb_from_xy_and_brightness` to the `Device` base class.

I will leave it to you to either choose a prefered direction or refactor it yourself.

Regards, Jan-Jaap

**Comment by stas-demydiuk** [comment]
@JanJaapKo I propose to extract `get_color_value` and `get_rgb_from_xy_and_brightness` and other common logic (if any) to own class inherited from `Device` class and then inherit 'color_light.py' and 'color_colortemp.py' from this new class

File: adapters/rgb_adapter.py [hunk 1]
```diff
@@ -36,16 +36,23 @@ def handleCommand(self, alias, device, device_data, command, level, color):
...
             red = colorObject['r']
             blue = colorObject['b']
+            ttime = 1
 
-            return {
-                'topic': device_data['friendly_name'] + '/set',
-                'payload': json.dumps({
+            payload = json.dumps({
                     "state": "ON",
+              # Disabled the transition time for now because hue bulb/zigbee2mqtt will
```
Comment (stas-demydiuk) [inline adapters/rgb_adapter.py]: @JanJaapKo what is the purpose of changing this file: as I see nothing was changed eventually?

**Comment by JanJaapKo** [comment]
@stas-demydiuk  I made a base class for color/color temp devices, from which the existing color_light and color_colortemp_light inherit. I've tested the color_light, works fine. @d-EScape would you be able to test the colortemp version on your Hue's?

If needed you can pull this version from my git: https://github.com/JanJaapKo/domoticz-zigbee2mqtt-plugin

**Comment by stas-demydiuk** [comment]
Thank you for the PR!

**Pull request merged**